### ISSUE TITLE
Version 4.0.1: Unable to adjust the size of a chart in a dashboard when the chart touches the right edge of the dashboard

### ISSUE THREAD

**Issue by Kim-frau** [apache/superset#29173]
### Bug description

Unable to adjust the size of a chart in a dashboard when the chart touches the right edge of the dashboard.
The only solution seems to delete the chart in the dashboard and import it again.

### How to reproduce the bug

1. Go to a dashboard
2. Import a graph
3. Adjust the width to the maximum on the right size of the dashboard
4. Impossible to make it smaller then

### Superset version

4.0.1

### Python version

3.9

### Node version

16

### Browser

Chrome

### Checklist

- [x] I have searched Superset docs and Slack and didn't find a solution to my problem.
- [x] I have searched the GitHub issue tracker and didn't find a similar bug report.
- [x] I have checked Superset's logs for errors and if I found a relevant Python stacktrace, I included it here as text in the "additional context" section.

**Comment by sebint** [comment]
Yes unable to resize if it is stretched to right most side of dashboard. But if we try to drag from the right bottom corner the drag handler appears. @Kim-frau,  have you got any solution?

**Comment by rusackas** [comment]
@justinpark might have some ideas on this one.

**Comment by rusackas** [comment]
Haven't heard any feedback or updates on this. Not sure if anyone's still facing this, but I've been resizing things all over in 4.1.1/5.0.0 without any significant issues. If you can provide updated context, we're happy to revisit/reopen. For now, calling it stale/inactionable.
